app/connectors/util/CustomHttpReader.scala

```diff
@@ -27,7 +27,8 @@ object CustomHttpReader extends HttpReads[HttpResponse] with HttpErrorFunctions
         s"\n  ${x._1} : ${x._2}"
     }}")
     response.status match {
-      case UNAUTHORIZED | BAD_GATEWAY => recode(INTERNAL_SERVER_ERROR, response)
+      case UNAUTHORIZED => recode(INTERNAL_SERVER_ERROR, response)
```

Comment:
I think we can remove this. The arrivals and departures backends check for 400 and 403 status codes.

Reply:
I'd leave it in, the issue is that THIS service has an issue - basically that it was unable to provide the bearer token. If we send it back up, we lose that context

Reply:
So would it be fine if I leave it as it is?

Reply:
Yes. Rob is correct. Logging could be added here but I think it needs to be added for when we revise the logging for all services.

Reply:
Okay then I'll leave it as it is.